src/main/java/stepdefs/ScenarioHooks.java

```diff
@@ -0,0 +1,23 @@
+package stepdefs;
+
+import core.LocalCreator;
+import core.RunLocationCreator;
+import core.WebDriverSingleton;
+import cucumber.api.java.After;
+import cucumber.api.java.Before;
+
+
+public class ScenarioHooks {
+    private RunLocationCreator runLocationCreator;
+
+    @Before
+    public void startBrowser() {
+        runLocationCreator = new LocalCreator();
```

Comment:
it should be set only one time before test run. In your case it will be done before each test